integration/test/cloudwatchlogs/resources/write_and_rotate_logs.py

```diff
@@ -0,0 +1,23 @@
+import json
```

Comment:
Can this be written in go? One of the goals of moving away from our current tests to keep single language tests. Like before we had java test but a golang code base now we want golang tests with the golang code base.

Reply:
Appreciate the context. I think to get this through the door, I'll hold off on trying to reinvent the wheel in Go, but might revisit this when I free up.